src/client/dfuse/dfuse_fuseops.c

```diff
@@ -88,16 +87,12 @@ dfuse_fuse_init(void *arg, struct fuse_conn_info *conn)
 	conn->want |= FUSE_CAP_READDIRPLUS;
 	conn->want |= FUSE_CAP_READDIRPLUS_AUTO;
 
-	conn->time_gran = 1;
-
-	if (dfuse_info->di_wb_cache)
-		conn->want |= FUSE_CAP_WRITEBACK_CACHE;
-
 #ifdef FUSE_CAP_CACHE_SYMLINKS
 	conn->want |= FUSE_CAP_CACHE_SYMLINKS;
 #endif
 	dfuse_show_flags(dfuse_info, conn->capable, conn->want);
 
+	conn->time_gran            = 1;
```

Comment:
IIRC this has something to do with ms or ns time?

Reply:
Yes. This isn't a logic change, just grouping the conn->want code together makes it appear new.

Reply:
Ah, I missed that